docs/_documentations/remotedeploy-projects-eclipse.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: docs
+title: Eclipse
+description: Creating and importing projects
+keywords: users, projects, Kubernetes, LDAP, user management, access management, login, deployment, pod, security, securing cloud connection, remote deployment of Codewind
+duration: 5 minutes
+permalink: remotedeploy-projects-eclipse
+type: document
+---
+
+# Creating and importing projects
+
+When you configure Codewind to be used remotely, you are ready to create a project that you can build and run remotely. 
+
+If you have projects that currently exist on your local computer that you would like to continue to develop remotely, follow [Import an existing local project to the cloud](##import-an-existing-local-project-to-the-cloud).
+
+## Create a new project
+
+Go to the Codewind pane and click the Plus (+) icon on the new cloud deployment:
```

Comment:
Go to the `Codewind Explorer` view, right-click on your remote connection and select `Create New Project`.